docs/installing/guided.rst

```diff
@@ -91,61 +94,66 @@ To run it, execute archinstall as a Python module:
+| bootloader           | systemd-bootctl/grub-install                        | Bootloader to be installed                                                           | Yes                                           |
++----------------------+-----------------------------------------------------+--------------------------------------------------------------------------------------+-----------------------------------------------+
+| custom-commands      | [ <command1>, <command2>, ...]                      | Custom commands to be run post install                                               | No                                            |
++----------------------+-----------------------------------------------------+--------------------------------------------------------------------------------------+-----------------------------------------------+
+| !encryption-password | any                                                 | Password to encrypt disk, not encrypted if password not provided                     | No                                            |
++----------------------+-----------------------------------------------------+--------------------------------------------------------------------------------------+-----------------------------------------------+
+| filesystem           | ext4 / btrfs / fat32 etc.                           | Filesystem for root and home partitions                                              | Yes                                           |
++----------------------+-----------------------------------------------------+--------------------------------------------------------------------------------------+-----------------------------------------------+
+| gfx_driver           | - "VMware / VirtualBox (open-source)"               | Graphics Drivers to install                                                          | No                                            |
+|                      | - "Nvidia"                                          |                                                                                      |                                               |
+|                      | - "Intel (open-source)"                             |                                                                                      |                                               |
+|                      | - "AMD / ATI (open-source)"                         |                                                                                      |                                               |
+|                      | - "All open-source (default)"                       |                                                                                      |                                               |
```

Comment:
This diff here is a bit unruly, a lot of changes to white space. The only substantive change I made to the table was to put all possible values for `gfx_driver` on separate lines. This makes the column "Values/Description" much shorter and doesn't hide the "Description" column anymore. The forth column "Required" is still hidden though. I added a note at the bottom of the table about that. Obviously this change makes the row for `gfx_driver` much taller, let me know if you think that's worse than before.